package.json

```diff
@@ -99,12 +100,15 @@
   },
   "man": "",
   "scripts": {
+    "build": "rm -rf dist && tsc",
+    "postbuild": "copyfiles -u 1 lib/**/*.ejs dist",
```

Comment:
I recommend adding [zx](https://github.com/google/zx) library from Google. It could be a game changer for us in the future if we require both zsh and JavaScript at some point in future.

Also, this way we don't need extra package to copy files

👀 Just a friendly suggestion - it might be worth considering. Feel free to ignore if you think it's an overkill

Reply:
Will look into it as part of improving DX.